apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -237,6 +254,11 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
         if (!get(processedRow, ['addons', 'meterResourceId'])) errors.push(MeterResourceNotFoundMessage)
         if (!get(processedRow, ['addons', 'valuesAmount'])) errors.push(NoValuesErrorMessage)
 
+        if (!validateMeterValue(get(processedRow, ['addons', 'value1']))) errors.push(MeterValueInvalidMessage.replace('{columnName}', Value1ColumnMessage))
```

Comment:
You can use parametrized message like this:
<img width="888" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/47c2baa9-52e3-41f8-827a-ca4d214b3fcd">

Reply:
Yes, we use both. This one looks more compact.

Reply:
But translation template provide more abstract syntax, since you can pass components there, use numbers, dynamic parameters based on locale. In other words Intl is translation tool we use, so it's best practice to pass parameters like this. (If we we switch this tool somehow this will probably can be uncaught), since it's using Intl format template without Intl itself

Reply:
Ok, refactored it :)